packages/js/test/plugins/auctionHouseModule/createAuctionHouse.test.ts

```diff
@@ -143,3 +144,61 @@ test('[auctionHouseModule] create new Auction House with SPL treasury', async (t
     },
   });
 });
+
+test('[auctionHouseModule] create new Auction House with Auctioneer', async (t: Test) => {
+  // Given we have a Metaplex instance.
+  const mx = await metaplex();
+
+  const auctioneerAuthority = Keypair.generate();
+
+  // When we create a new Auction House with minimum configuration.
```

Comment:
Please ensure the comments are updated when copy/pasting tests.

```suggestion
  // When we create a new Auction House with an Auctioneer authority.
```